java/scripts/sign_and_bundle_release.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ];
+then
+      echo -e "Usage: $0 \e[4mVERSION\e[0m"
+      echo -e "where \e[4mVERSION\e[0m is part of the tag release/java/<version>, ex v1.0.0"
```

Comment:
List of requirements might be nice. AFAICT it's `jq`, `curl`, `wget`, `gpg`, `cosign`, `{md5,sha1,sha256,sha512}sum`, `jar`